python/ray/util/sgd/v2/trainer.py

```diff
@@ -444,6 +409,90 @@ def to_tune_trainable(self, train_func: Callable[[Dict[str, Any]], T]
                                       self._num_workers, self._use_gpu,
                                       self._resources_per_worker)
 
+    def to_workers(self, train_cls: Type, *args, **kwargs) -> "SGDWorkerGroup":
```

Comment:
nit: Should we rename this to `to_worker_group`?

Reply:
Sure we can do that

Reply:
Updated